tests/e2e/odh_manager_test.go

```diff
@@ -27,17 +27,4 @@ func (tc *testContext) validateOwnedCRDs(t *testing.T) {
 
 	require.NoErrorf(t, tc.validateCRD("dscinitializations.dscinitialization.opendatahub.io"),
 		"error in validating CRD : dscinitializations.dscinitialization.opendatahub.io")
-
```

Comment:
can we move these 4 into https://github.com/opendatahub-io/opendatahub-operator/blob/main/tests/e2e/dsc_creation_test.go#L340